static/templates/template/container.html

```diff
@@ -1,8 +1,8 @@
-<md-subheader class="md-no-sticky">Use our Task Builder to design your task. Arrange and customize your task using our draggable template items. Double click a component to modify its properties.</md-subheader>
+<md-subheader class="md-no-sticky" ng-init="project.getLastMilestoneComments()">Use our Task Builder to design your task. Arrange and customize your task using our draggable template items. Double click a component to modify its properties.</md-subheader>
```

Comment:
@shirishgoyal can we load it once when we load the controller, i can't think of too many instances when a reviewer won't want to see the comments.